docs/release_notes/v0.7.0.md

```diff
@@ -0,0 +1,237 @@
+
+**Added** Sendgrid binding spec
+
+**Added** Readiness configuration for Kubernetes
+
+**Improved** application insights distributed tracing onboarding guide by uploading localfowarder docker image to [dapr official dockerhub](https://hub.docker.com/r/daprio/dapr-localforwarder)
+
+
+## Tests
+
+**Added** Performance test for service invocation
+
+**Added** Performance test configuration and test app using Fortio
+
+**Improved** Coverage for service invocation
+
+**Added** Message analyzer delay monitor for Long Haul
+
+**Fixed** Containers not updating if config YAML doesn't change
+
+**Improved** Long haul tests validation and monitoring
+
+**Added** Long haul test snapshot monitor
+
+**Added** Long haul test hashtag counter delay monitor
+
+**Added** Long haul test Generic error spike monitor
+
+**Added** Long haul test feed stream generator
+
+**Added** Hashtag snapshot service
+
+
+## Upgrading to Dapr 0.7.0
+
+If you're upgrading from an older version of Dapr to 0.7.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have:
+
+```bash
+dapr uninstall --all
+```
+
+Next, get the latest CLI following [these](https://github.com/dapr/cli#installing-dapr-cli) instructions, or alternatively download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have downloaded the CLI, run:
+
+```bash
+dapr init
+```
+
+Wait for the update to finish, and you're good to go!
+Make sure you have the right runtime version (0.7.0) with:
+
+```bash
+dapr --version
+```
+
+### Kubernetes
+
+Download the latest CLI release as outlined above in the `Local Machine / Self-hosted` section.
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+```bash
+dapr uninstall --kubernetes
+```
+
+It's fine to ignore any errors that might show up.
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+```bash
+helm repo update
+```
+
+Wait until Dapr is uninstalled, and make sure the `dapr-system` namespace is gone.
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, if you want to install Dapr in a dev/test setup, run:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+After Dapr 0.7.0 has been installed, perform a rolling restart for your deployments.
+
+## Breaking Changes
+
+### gRPC Service Invocation
+
+### Tracing configuration
```

Comment:
With 0.7.0 release, Dapr trace configuration is simplified. User now has to provide "samplingRate" to enable or disable the traces.  Set "samplingRate: "0" to disable the traces. Set "samplingRate: "1" to  always generate the the traces. Refer https://github.com/dapr/docs/blob/master/concepts/observability/traces.md for more details.